### PR TITLE
mirror minikube artifacts on S3 too

### PR DESCRIPTION
**Pull request by alban**

### CODE REVIEW

File: 00-Getting_Started/start_minikube.sh [hunk 1]
```diff
@@ -5,10 +5,14 @@ set -e
 # Download the executable and the ISO
 echo Downloading minikube components
 if [[ ! -f minikube ]]; then
-  wget https://eleven.dev.kinvolk.io/u/alban/cloud-native-bpf-workshop/minikube
+  URL=https://eleven.dev.kinvolk.io/u/alban/cloud-native-bpf-workshop/minikube
+  URL=https://2020-08-17-cloud-native-bpf-workshop-public.s3.eu-central-1.amazonaws.com/minikube
+  time wget $URL
```
Comment (marga-kinvolk) [inline 00-Getting_Started/start_minikube.sh:10]: I'm guessing this was a test, will you remove the extra URL and time?

Comment (alban) [reply 00-Getting_Started/start_minikube.sh]: Updated :)

File: 00-Getting_Started/start_minikube.sh [hunk 2]
```diff
@@ -18,7 +20,17 @@ sha256sum -c <<EOF
...
+
+if [[ ! $REPLY =~ ^yes$ ]]; then
+  echo Abort.
+  exit 1
+fi
+
 echo Executing minikube
 chmod +x minikube
 ./minikube delete
 ./minikube start --driver=kvm2 --iso-url=file://$(pwd)/minikube.iso
```
Comment (marga-kinvolk) [inline 00-Getting_Started/start_minikube.sh:35]:
```suggestion
echo Verifying that kernel headers are correctly installed in minikube
```

File: 00-Getting_Started/start_minikube.sh [hunk 3]
```diff
@@ -18,7 +20,17 @@ sha256sum -c <<EOF
...
+./minikube ssh -- 'bash -c "uname -a ; if [ -e /sys/kernel/kheaders.tar.xz ] ; then echo kheaders.tar.xz available ; else echo kheaders.tar.xz missing ; fi"'
```
Comment (marga-kinvolk) [inline 00-Getting_Started/start_minikube.sh:36]:
```suggestion
echo Verifying that kernel headers are correctly installed in minikube
./minikube ssh -- 'bash -c "uname -a ; if [ -e /sys/kernel/kheaders.tar.xz ] ; then echo kheaders.tar.xz available ; else echo kheaders.tar.xz missing ; fi"'
```

**Pull request merged**